src/features/features.controller.ts

```diff
@@ -0,0 +1,104 @@
+import {
+    Controller,
+    Get,
+    Post,
+    Body,
+    Patch,
+    Param,
+    Delete,
+    ParseIntPipe,
+    UseGuards,
+    Request,
+    HttpException,
+    HttpStatus,
+    NotFoundException,
+} from "@nestjs/common";
+import { FeaturesService } from "./features.service";
+import { CreateFeatureDto } from "./dto/create-feature.dto";
+import { UpdateFeatureDto } from "./dto/update-feature.dto";
+import { ApiBearerAuth, ApiCreatedResponse, ApiTags } from "@nestjs/swagger";
+import { Feature } from "./entities/feature.entity";
+import { JwtAuthGuard } from "src/auth/jwt-auth.guard";
+
+@Controller("teams")
```

Comment:
Based on Jenny wanting the path to start with the module that owns it I added teams to the controller since it doesn't affect the Swagger @ApiTag but I can remove it and just add to each path if that's preferable. 
![image](https://github.com/chingu-x/chingu-dashboard-be/assets/104289709/3f43db86-0c9d-4e92-95e0-766407a7e99d)

Curious, my understanding is that the @Controller() just controls what prefixes the path, does it affect things elsewhere as well?

Reply:
yeah i think so, it's only the path

Reply:
Okay, I'm going to leave as is then since all the paths are prefixed with teams.